templates/terraform/environments/prod/user_access.tf

```diff
@@ -115,4 +189,16 @@ locals {
       ]
     }
   ]
+
+  # define Kubernetes policy for deployer
+  k8s_deployer_access = [
+    {
+      verbs      = ["exec", "create", "list", "get", "delete", "patch", "update"]
+      api_groups = ["*"]
+      resources = ["deployments", "configmaps", "pods", "pods/exec", "pods/log", "pods/status", "pods/portforward",
```

Comment:
Let's tweak this a bit to remove port forward and exec, and maybe make secrets write-only? 
Please take a look at the default deployment manifests to make sure this policy covers it. Otherwise even our demo apps won't be able to deploy. https://github.com/commitdev/zero-deployable-backend/tree/main/templates/kubernetes
(For example, one thing I think we create is a PodDisruptionBudget

Reply:
added "watch", set secrets write-only, removed portforward/exec etc.